packages/react/src/React.js

```diff
@@ -105,7 +105,7 @@ export {
   useCallback,
   useContext,
   useEffect,
-  useEvent as experimental_useEvent,
+  useEffectEvent as experimental_useEffectEvent,
```

Comment:
Seeing the others exports using the `unstable_` prefix, does `experimental_` has a special meaning here?